Ozempic_AI_Webapp/src/theme/tokens.css

```
@@ -0,0 +1,66 @@
+  --space-md: 16px;
+  --space-lg: 24px;
+  --space-xl: 32px;
+
+  /* Radius */
+  --radius-sm: 12px;
+  --radius-md: 18px;
+  --radius-lg: 24px;
+  --radius-hero: 32px;
+
+  /* Typography */
+  --font-display: 'Fraunces', ui-serif, 'New York', Georgia, serif;
```

Stylelint (17.11.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)